### PR TITLE
Revise PriorBox Spec

### PR DESCRIPTION
**Pull request by blesniewski**
### Details:
 - *Cosmetic changes: moved detailed description up, bolded some keywords, removed some braces, added smaller newline breaks between formulas in tex.*

### Tickets:
 - *37462*
 - *45986*

### CODE REVIEW

File: docs/ops/detection/PriorBox_1.md [hunk 1]
```diff
@@ -82,80 +116,36 @@
...
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 **Inputs**:
 
-*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. Required.
+*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. **Required**.
```
Comment (jdanieck) [inline docs/ops/detection/PriorBox_1.md:141]: Please introduce *Types* section and refer to it from inputs/outputs section. We need to be consistent with the rest of spec.

File: docs/ops/detection/PriorBox_1.md [hunk 2]
```diff
@@ -82,80 +116,36 @@
...
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 * *fixed_size*
 
     * **Description**: *fixed_size* is an initial box size (in pixels). For example, *fixed_size* equal to 15 means that the initial box size is 15.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
```
Comment (jdanieck) [inline docs/ops/detection/PriorBox_1.md:127]: Removal of backticks should be reverted and other attributes should be re-checked to introduce backticks.  @ggalieroc please confirm that nothing has changed recently and we should use backticks for types in attributes section.

Comment (ggalieroc) [reply docs/ops/detection/PriorBox_1.md:127]: Correct, the type section of attributes is surrounded by backticks

File: docs/ops/detection/PriorBox_1.md [hunk 3]
```diff
@@ -82,80 +116,36 @@
 
     * **Description**: *fixed_ratio* is an aspect ratio of a box. For example, *fixed_ratio* equal to 2.000000 means that the aspect ratio for the first box aspect ratio is 2.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
```
Comment (jdanieck) [inline docs/ops/detection/PriorBox_1.md:120]: Not required attribute must have default value. Please go through all attributes and check correctness of `Default value`.

**Review by jdanieck — APPROVED** [review]

File: docs/ops/detection/PriorBox_1.md [hunk 4]
```diff
@@ -83,78 +117,39 @@
...
-    ymin = \frac{\frac{center_y - s}{2}}{H}
-    \f]
-    \f[
-    xmax = \frac{\frac{center_x + s}{2}}{W}
-    \f]
-    \f[
-    ymin = \frac{\frac{center_y + s}{2}}{H}
-    \f]
+* *T_INT*: any supported integer type.
+* *T_OUT*: floating point (`fp32`) type.
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:152]: remove mentioning fp32 type. Plugins may use other types

File: docs/ops/detection/PriorBox_1.md [hunk 5]
```diff
@@ -40,31 +74,31 @@
...
     * false or 0 - clipping is not performed
     * true or 1 - each value in the output tensor is clipped to [0,1] interval.
-  * **Type**: boolean
+  * **Type**: `boolean`
   * **Default value**: false
   * **Required**: *no*
 
 * *step*
 
   * **Description**: *step* is a distance between box centers. For example, *step* equal 85 means that the distance between neighborhood prior boxes centers is 85.
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:83]: Please, remove all `For example, *XXX* equals to ...`. These statements are useless in most cases. If they really make sense the update the first sentence in the description to use it.

File: docs/ops/detection/PriorBox_1.md [hunk 6]
```diff
@@ -6,21 +6,55 @@
...
+    \f]
+    \f[
+    ymin = \frac{\frac{center_y - s}{2}}{H}
+    \f]
+    \f[
+    xmax = \frac{\frac{center_x + s}{2}}{W}
+    \f]
+    \f[
+    ymin = \frac{\frac{center_y + s}{2}}{H}
+    \f]
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:41]: There is no mentioning of `clip` attribute. I expect to see it in the detailed description.

File: docs/ops/detection/PriorBox_1.md [hunk 7]
```diff
@@ -40,31 +74,31 @@
   * **Range of values**:
     * false or 0 - clipping is not performed
     * true or 1 - each value in the output tensor is clipped to [0,1] interval.
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:76]: Wrap all interval values with back ticks

File: docs/ops/detection/PriorBox_1.md (hunk 6 above)
Comment (lazarevevgeny) [reply docs/ops/detection/PriorBox_1.md:41]: I don't see how `aspect_ratio` and `flip` attributes are used as well

File: docs/ops/detection/PriorBox_1.md [hunk 8]
```diff
@@ -83,78 +117,39 @@
...
-*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. Required.
+*   **1**: `output_size` - 1D tensor of type *T_INT* with two elements `[height, width]`. Specifies the spatial size of generated grid with boxes. **Required**.
 
-*   **2**: `image_size` - 1D tensor with two integer elements `[image_height, image_width]` that specifies shape of the image for which boxes are generated. Required.
+*   **2**: `image_size` - 1D tensor of type *T_INT* with two elements `[image_height, image_width]` that specifies shape of the image for which boxes are generated. **Required**.
 
 **Outputs**:
 
-*   **1**: 2D tensor of shape `[2, 4 * height * width * priors_per_point]` with box coordinates. The `priors_per_point` is the number of boxes generated per each grid element. The number depends on layer attribute values.
+*   **1**: 2D tensor of shape `[2, 4 * height * width * priors_per_point]` and type *T_OUT* with box coordinates. The `priors_per_point` is the number of boxes generated per each grid element. The number depends on layer attribute values.
```
Comment (lazarevevgeny) [inline docs/ops/detection/PriorBox_1.md:147]: Do not use word `layer`. Use `operation` instead

File: docs/ops/detection/PriorBox_1.md (hunk 6 above)
Comment (blesniewski) [reply docs/ops/detection/PriorBox_1.md:41]: Made sure to mention `clip`, however I don't think that it's necessary to provide pseudo-code for the whole operation. Reference implementation is quite readable for that purpose.

**Review by ggalieroc — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**